install_config/configuring_authentication.adoc

```diff
@@ -1665,8 +1665,8 @@ with a non-empty value is used. The
 link:http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims[standard claims] are:
 
 [horizontal]
-`sub`:: The user identity.
-`preferred_username`:: The preferred user name when provisioning a user.
+`sub`:: The ID of the user within the configured identity provider. For example `user1`. This can be created before the authorization process. When added to the identity object, this is prefixed by the identity provider object. For example `myidp:user1`. If the identity object does not exist, so, if the user has not been created, then the prefix depends on the `preferred_username`.
+`preferred_username`:: The preferred user name when provisioning a user if the `sub` value is not present.
```

Comment:
Revert this change, this is a description of the claims, not of precedence relative to other claims. The configuration they specify determines the fallback order